docs/document/docs/components/desktop/table.md

```diff
@@ -85,8 +85,8 @@ Here is a list of properties that can be used for modifying component style:
-| --kuc-table-action-button-right | <ul><li>This property is used to set the offset of the right edge of the action button column from the right edge of Table's nearest scrolling ancestor</li><li>It works when `actionButtonPosition` is right</li><li>For example, when a child table is set up in a table and `actionButtonPosition` is right for both the child table and the parent table</li><ul><li>You can set the width of the action button column of the parent table to `--kuc-table-action-button-right` in the child table to avoid the problem that the action button columns may overwrite each other</li><li>In general, it is set to 77px if both the add/remove row buttons of the parent table are shown, or 41px if only one of the add/remove row buttons of the parent table is shown</li></ul></ul> |
+| --kuc-table-action-button-left | <ul><li>This property is used to set the offset of the left edge of the add/remove row button column from the left edge of Table's nearest scrolling ancestor</li><li>It works when `actionButtonPosition` is left</li><li>For example, when a child table is set up in a table and `actionButtonPosition` is left for both the child table and the parent table</li><ul><li>You can set the width of the add/remove row button column of the parent table to `--kuc-table-action-button-left` in the child table to avoid the problem that the add/remove row button columns may overwrite each other</li><li>In general, it is set to 77px if both the add/remove row buttons of the parent table are shown, or 41px if only one of the add/remove row buttons of the parent table is shown</li></ul></ul> |
+| --kuc-table-action-button-right | <ul><li>This property is used to set the offset of the right edge of the add/remove row button column from the right edge of Table's nearest scrolling ancestor</li><li>It works when `actionButtonPosition` is right</li><li>For example, when a child table is set up in a table and `actionButtonPosition` is right for both the child table and the parent table</li><ul><li>You can set the width of the add/remove row button column of the parent table to `--kuc-table-action-button-right` in the child table to avoid the problem that the add/remove row button columns may overwrite each other</li><li>In general, it is set to 77px if both the add/remove row buttons of the parent table are shown, or 41px if only one of the add/remove row buttons of the parent table is shown</li></ul></ul> |
```

Comment:
> 41px if only one of the add/remove row buttons of the parent table is shown

User wanna display add and remove row button fully in most cases, so considering that, I think we don't need this "only one" pattern. What do you think?

Reply:
When there is one row, only the add button is shown, in which case it needs to be set to 41px;
When there are multiple rows, both the add/delete buttons are shown, in which case it needs to be set to 77px.

Reply:
Ok, you assume that developer wanna change the px following the change of number of the row.
I think it's ok to keep the current description. Thanks for the explanation.